ArduCopter/GCS_Mavlink.cpp

```diff
@@ -77,6 +77,36 @@ MAV_STATE GCS_MAVLINK_Copter::vehicle_system_status() const
 }
 
 
+void GCS_MAVLINK_Copter::send_attitude_target()
+{
+#if MODE_GUIDED_ENABLED == ENABLED
+    if (!copter.flightmode->in_guided_mode() ||
+        copter.mode_guided.submode() != ModeGuided::SubMode::Angle) {
+        return;
+    }
+
+    Attitude_Target_Info target;
+    target.thrust = 0.0;
+    copter.mode_guided.get_attitude_target(target);
```

Comment:
```suggestion
    copter.mode_guided.get_target_attitude(target);
```

... to match all the other callbacks in there....

Reply:
I can do that.  
I initially made the name to match the mavlink message name but think the structure has more use than just for mavlink.

Reply:
It's a bit funny that the get_attitude_target is in the mode class but we don't actually call flightmode->get_attitude_target() instead we directly ask guided mode.

How about we simplify a bit and implement get_attitude_target (or get_target_attitude) within mode.cpp because there's nothing guided mode specific about it?  Then we can remove the checks for whether we're in guided mode and maybe even just return three arguments and avoid the AttitudeTargetInfo structure.

Reply:
We only accept Set_Attitude_Target in guided mode though?  We could allow someone to get all of the attitude target information no matter the mode?

You are correct i could simplify it all to just  `GCS_MAVLINK_Copter::send_attitude_target()`.

Reply:
yes, I think the attitude target could always be provided.  Sometimes it would come from the external caller and other times it might come from the waypoint navigation library, etc.